chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -347,6 +347,31 @@ def make_zone(zone)
 
 # We would like to bind service only to ip address from admin network
 admin_addr = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address
+# if we're the admin node and no other node exists yet just listen on loop back
```

Comment:
Oops, that's completely bogus anyway and from different attempt to fix the problem. I'll rephrase.